docs/products/tools/api/get-started/index.md

```diff
@@ -0,0 +1,89 @@
+---
+
+The Linode API allows you to automate any task that can be performed by the Cloud Manager. This includes creating and managing all of our products (like Compute Instances and Managed Databases), your customer account (like adding new users or viewing invoices), and even creating support tickets. As an example of an API call, the command below deploys a new 2GB Compute Instance in the Newark data center using the Debian 11 image.
+
+```command
+curl -X POST https://api.linode.com/v4/linode/instances \
+    -H "Authorization: Bearer $TOKEN" -H "Content-type: application/json" \
+    -d '{"type": "g6-standard-2", "region": "us-east", "image": "linode/debian11", "root_pass": "[password]", "label": "[label]"}'
+```
+
+This guide helps you get set up to run this example. Note that if you run this command, you create and are charged for a 2GB Compute Instance. See [Linode Pricing](https://www.linode.com/pricing/) for details on these charges.
+
+## Get an Access Token
+
+Authorization for the Linode API is handled through personal access tokens. Personal access tokens grant permissions to read or write to an API endpoint. All API requests to non-public resources must be authenticated with an access token.
+
+{{< note >}}
+If you are building an application which needs to authenticate multiple users (for example, a custom interface to Linode's infrastructure for your organization), you can set up an [OAuth authentication flow](/docs/api/#oauth) to generate tokens for each user.
+{{< /note >}}
+
+For full instructions on creating and managing personal access tokens, see [Manage Personal Access Tokens](/docs/products/tools/api/guides/manage-api-tokens/#create-an-api-token).
+
+1. Log into the Cloud Manager.
+
+1. Click on your username at the top of the screen and select **My Profile**.
+
+1. Select the **API Tokens** tab:
+
+1. Click on **Create Personal Access Token** and choose the access rights you want users authenticated with the new token to have.
+
+1. When you have finished, click **Create** to generate an API token string. Copy the token and save it in a secure location. **You will not be able to view the token through the Cloud Manager after closing the popup.**
+
+## Authenticate Requests
+
+Once you have created a personal access token, it needs to be sent along in the header of most API requests. The header should use the format:
+
+```command
+Authorization: Bearer <token-string>
+```
+
+Store the token as a temporary shell variable to simplify repeated requests. Replace `<token-string>` in this example:
+
+```command
+export TOKEN=<token-string>
+```
+
+## Get Configuration Parameters
+
+Specify the type, region, and image for the new Linode.
+
+1. Review the list of available images:
+
+    ```command
+    curl https://api.linode.com/v4/images/ | json_pp
```

Comment:
We use json_pp a lot in these instructions, do we have any language stating that you'll have to download and install this package?